index.html

```diff
@@ -250,26 +250,23 @@ <h2>
     </section>
     
     </div>
-    <section class="bg_color">
+    <section class="bg_color">      
       
-      
-      <div class="contact_section">
-        
+      <div class="contact_section">        
        
-        <div class="contact_form">
-          
+        <div class="contact_form">          
           <header>
             <h3>Contact me</h3>
           </header>
           <div class="text3">
             
             <h2>
-              If you have an application you are interested in developing, a
-            feature that you need built or a project that needs coding. I’d love
+              If you have an application you are interested in developing, 
+            features that you need built or a project that needs coding. I’d love
             to help with it.
             </h2>
           </div>
-        <form     
+          <form     
           action="https://formspree.io/f/xpznodvr"
           method="post"
```

Comment:
- [ ] Kindly adjust the padding of your inputs to match the original design.

![image](https://user-images.githubusercontent.com/98453979/207375711-27da6201-25ab-4350-945b-b1825948c466.png)


### Checkout original design

![image](https://user-images.githubusercontent.com/98453979/207375800-dab2d03a-794e-4961-bc05-ab500e6f9962.png)